### PR TITLE
📝 docs: add `read*` functions with cleanup

### LINKED ISSUE #82: docs for getSecondsToFinalizable

### PR DESCRIPTION
**Pull request by Sabnock01**
The following actions have been added to the docs:

* `getSecondsToFinalizable`
* `readFinalizedWithdrawals`
* `readProvenWithdrawals`

Along with some other minor cleanup.

Resolves #82

### CODE REVIEW

File: site/docs/actions/public/L1/getSecondsToNextL2Output.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 Returns how long until the next L2 output, for a given chain, is posted on L1. This is useful when waiting to prove a withdrawal.
 
-```ts
+::: code-group
+
+```ts [example.ts]
```
Comment (wilsoncusack) [inline site/docs/actions/public/L1/getSecondsToNextL2Output.md:7]: we only really do this if we have another tab called like `config.ts`

File: site/docs/actions/public/L1/readFinalizedWithdrawals.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### l2chain
+
+- **Type:** `OpStackChain`
+
+The L2 chain to deposit to.
+
+### withdrawalHash
+
+- **Type:** `0x{string}`
```
Comment (wilsoncusack) [inline site/docs/actions/public/L1/readFinalizedWithdrawals.md:40]:
```suggestion
- **Type:** `Hex`
```

Comment (wilsoncusack) [reply site/docs/actions/public/L1/readFinalizedWithdrawals.md:40]: OR we could do like https://www.opviem.sh/docs/actions/public/L1/simulateProveWithdrawalTransaction.html#withdrawaltransaction

`MessagePassedEvent`[`withdrawalHash`]

File: site/docs/actions/public/L1/readProvenWithdrawals.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# readProvenWithdrawals
+
+Returns a `ProvenWithdrawal` struct containing the `outputRoot`, `timestamp`, and `l2OutputIndex` for a given withdrawal hash. Returns error if withdrawal has not been proven.
```
Comment (wilsoncusack) [inline site/docs/actions/public/L1/readProvenWithdrawals.md:3]: We should make a TODO to put ProvenWithdrawal in types/glossary so we can point to it from multiple places

File: site/docs/actions/public/L1/getSecondsToNextL2Output.md (hunk 1 above)
Comment (Sabnock01) [reply site/docs/actions/public/L1/getSecondsToNextL2Output.md:7]: Seen this present in a lot of counterexamples like [getL2HashesForDepositTx](https://github.com/base-org/op-viem/blob/main/site/docs/actions/public/L1/getL2HashesForDepositTx.md) but will modify them all as needed.

File: site/docs/actions/public/L1/readFinalizedWithdrawals.md (hunk 2 above)
Comment (Sabnock01) [reply site/docs/actions/public/L1/readFinalizedWithdrawals.md:40]: I like that style.

File: site/docs/actions/public/L1/readProvenWithdrawals.md (hunk 3 above)
Comment (Sabnock01) [reply site/docs/actions/public/L1/readProvenWithdrawals.md:3]: Working on that in #115

File: site/docs/actions/public/L1/getSecondsToNextL2Output.md (hunk 1 above)
Comment (wilsoncusack) [reply site/docs/actions/public/L1/getSecondsToNextL2Output.md:5]: ah ok, sorry for the confusion then. That doc was written by someone else a while ago and I didn't QA closely. Not a huge deal either way. Thanks for eyeing the details

File: site/docs/actions/public/L1/getSecondsToFinalizable.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Parameters
+
+### l2chain (optional)
+
+- **Type:** `OpStackChain`
+
+The L2 chain to deposit to.
+
+### optimismPortalAddress (optional)
```
Comment (wilsoncusack) [inline site/docs/actions/public/L1/getSecondsToFinalizable.md:34]: sorry, this one actually takes two contract addresses, both of which are optional but must be passed if l2Chain not passed. https://github.com/base-org/op-viem/blob/main/src/actions/public/L1/getSecondsToFinalizable.ts#L29-L30

**Review by wilsoncusack — COMMENTED** [review]
Sorry @Sabnock01 one more thing and then I think we're good. Appreciate you!

File: site/docs/actions/public/L1/getSecondsToFinalizable.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- **Type:** [`Address`](https://viem.sh/docs/glossary/types#address)
+
+The `OptimismPortal` contract where the sendMessage call should be made. MUST be specified if [l2Chain](#l2chain-optional) not passed.
```
Comment (wilsoncusack) [inline site/docs/actions/public/L1/getSecondsToFinalizable.md:38]:
```suggestion
The address of the `OptimismPortal` contract where the `readProvenWithdrawals` call will be made. MUST be specified if [l2Chain](#l2chain-optional) not passed.
```

File: site/docs/actions/public/L1/getSecondsToFinalizable.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+### l2OutputOracleAddress (optional)
+
+- **Type:** [`Address`](https://viem.sh/docs/glossary/types#address)
+
+The address of the L2OutputOracle contract. MUST be provied if [l2Chain](l2chain-optional) is not.
```
Comment (wilsoncusack) [inline site/docs/actions/public/L1/getSecondsToFinalizable.md:44]:
```suggestion
The address of the L2OutputOracle contract where the `FINALIZATION_PERIOD_SECONDS` call will be made. MUST be provied if [l2Chain](l2chain-optional) is not.
```

**Comment by wilsoncusack** [comment]
@Sabnock01 I think there's a conflict with your other PR that went in

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**